main.go

```diff
@@ -4,22 +4,26 @@ import (
 	"encoding/json"
 	"flag"
 	"fmt"
-	"github.com/antonlindstrom/mesos_stats"
-	"github.com/golang/glog"
-	"github.com/prometheus/client_golang/prometheus"
 	"io/ioutil"
 	"net/http"
 	"os"
 	"time"
+
+	"github.com/antonlindstrom/mesos_stats"
+	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/prometheus"
 )
 
 var (
 	configFile  = flag.String("config.file", "", "Path to config file.")
 	metricsPath = flag.String("web.telemetry-path", "/metrics", "Path under which to expose metrics.")
+)
+
+var (
 	cpuLimitVal = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
 			Namespace: "mesos_task",
-			Name:      "cpus_limit",
+			Name:      "cpu_limit",
 			Help:      "CPU share limit.",
```

Comment:
Actually, do you know more about this metric? Is it CPU shares, or number of CPUs? It's called `cpusLimit` in Mesos, but I couldn't find anywhere quickly what it really means (relative shares vs. number of allowed cores). It would be great to clarify that and name the metric accordingly...